SPECS/ccache/ccache.spec

```
@@ -1,36 +1,48 @@
 Name:           ccache
 Summary:        Compiler Cache
-Version:        3.6
-Release:        2%{?dist}
+Version:        4.6
+Release:        1%{?dist}
 License:        BeOpen and BSD and GPLv3+ and (Patrick Powell's and Holger Weiss' license) and Public Domain and Python and zlib
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://ccache.dev
 Source0:        https://github.com/%{name}/%{name}/releases/download/v%{version}/%{name}-%{version}.tar.gz
 
-BuildRequires:  make
+BuildRequires:  cmake
 
 %description
 Ccache (or “ccache”) is a compiler cache. It speeds up recompilation by caching previous
 compilations and detecting when the same compilation is being done again.
 
 %prep
-%setup -q 
+%setup -q
 
 %build
-%configure
+mkdir build
+pushd build
+%cmake .. -DREDIS_STORAGE_BACKEND=OFF -DENABLE_TESTING=ON
 make %{?_smp_mflags}
+popd
 
 %install
-make install DESTDIR=%{buildroot}
+pushd build
+make DESTDIR=%{buildroot} install
```

Suggestion, since we're here:

```suggestion
%make_install
```

---

sure, updated